CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+40.0.0
+------
+
+- Upgrade aws-sdk to v2
```

Comment:
Should we add the breaking interface with ec2.metadata and add details that this applies to the cloudwatch backend + AWS cloudprovider?

Reply:
Yeah can do